File: app/src/main/java/can/siempredelao/f1kotlin/backend/StandingsTable.java
package can.siempredelao.f1kotlin.backend;

import com.google.gson.annotations.SerializedName;

import java.util.List;

public class StandingsTable {

    public String              driverId;
    public String              driverStandings;
    @SerializedName("StandingsLists")
    public List<StandingsList> standingsLists;

}

File: app/src/main/kotlin/can/siempredelao/f1kotlin/backend/model/StandingsTable.kt
package can.siempredelao.f1kotlin.backend.model

import com.google.gson.annotations.SerializedName

class StandingsTable(val driverId: String,
                     val driverStandings: String,
                     @SerializedName("StandingsLists") val standingsLists: List<StandingsList>)
